doc/source/user/absolute_beginners.rst

```diff
@@ -929,18 +960,30 @@ array. NumPy offers functions like ``ones()`` and ``zeros()``, and the
 All you need to do is pass in the number of elements you want it to generate::
 
   >>> np.ones(3)
+  array([1., 1., 1.])
   >>> np.zeros(3)
-  >>> np.random.random(3)  # the simplest way to generate random numbers
+  array([0., 0., 0.])
+  # the simplest way to generate random numbers
+  >>> np.random.default_rng().random(3)
```

Comment:
Am I right in thinking encourages users to create a new rng each time, which presumably is not a good idea?

Or does this return the same global instance each time?

Reply:
You are correct. The better usecase is line 983. Should we remove the ambiguity and show only the better version?

Reply:
I think so, yes. I think the line below is trying to show 2d arrays working too, and should otherwise use exactly the same approach as this section.

Reply:
ok